### PR TITLE
clientv3: update base example with TLS

### PR DESCRIPTION
**Pull request by gyuho**
For https://github.com/coreos/etcd/issues/6148.

### CODE REVIEW

File: clientv3/example_test.go [hunk 1]
```diff
@@ -29,9 +30,19 @@ var (
 )
 
 func Example() {
+	tlsInfo := transport.TLSInfo{
```
Comment (xiang90) [inline clientv3/example_test.go]: how about a separate TLSExample?

Comment (gyuho) [inline clientv3/example_test.go]: Sure, will fix with more detailed comments.

**Comment by gyuho** [comment]
@xiang90 PTAL. Thanks.

**Comment by xiang90** [comment]
lgtm

**Comment by eho-eho** [comment]
Thanks.

**Pull request merged**